prometheus/operator/base/cluster-metrics-servicemonitor.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: cluster-monitor-federation
+  labels:
+    monitor-component: cluster-resources
+    team: opendatahub
+spec:
+  endpoints:
+    - bearerTokenSecret:
+        key: token
+        name: odh.prometheus-k8s
+    - honorLabels: true
```

Comment:
There is a stray `-` here, there should only be one endpoint object

Reply:
thanks @4n4nd! :pray: